USING_WITH_PYRIGHT.md

```diff
@@ -0,0 +1,305 @@
+}
+```
+
+Where `pylanceStubs` contains the contents of the `bundled` and `native-stubs` folders.
+
+We have plans to make this easier in the [future](https://github.com/microsoft/pylance-release/discussions/3638).
+
+## Settings differences
+
+Both Pylance and Pyright have [settings](https://github.com/microsoft/pylance-release#settings-and-customization) to control behavior. For the most part, they use the same values, but the few differences do have an impact on the analysis.
+
+| Setting | Pylance default | Pyright default | Description | Potential Impact |
+|----|----|----|----|----|
+| python.analysis.autoSearchPaths | true | false|  Adds 'src' to the list of search paths | This may change what files are found when analyzing. So if you're getting missing imports for things in your 'src' tree, this might be why |
+| python.analysis.indexing | true | false | All files in the current virtual environment are parsed to a certain depth. This gives Pylance more information for auto import completions and other things. | This can have an impact on completions and hover information, but shouldn't impact errors returned.
+| python.analysis.useLibraryCodeForTypes | true | false | When true, this means 3rd party libraries are analyzed to produce type information. Without it, all 3 party libraries are just assumed to be of type Any unless they provided type information explicitly | This setting changes the types found by Pylance/Pyright. It can definitely cause differences in errors reported, so make this the same for both if you want consistent results. A lot of the time this is the sole cause of differences. |
+| python.analysis.typeCheckingMode | off | basic | Determines what diagnostics are shown. | Pylance defaults to off, but there is a possibility that VS code will force this to 'basic' for pylance users. If you want to guarantee this is the same as Pyright, force it to 'basic' (or whatever you want to enforce) by specifying it in your settings.json|
+| python.analysis.diagnosticMode | openFilesOnly | openFilesOnly | Determines which files are analyzed. `openFilesOnly` means only those files that are open in an IDE | You'll likely set this to 'workspace' for the CI version of Pyright. This means Pylance would likely not give you all of the same errors since it's only looking at open files. |
+| python.analysis.diagnosticSeverityOverrides | - | - | Allows a user to override the severity levels for individual diagnostic rules. | See below |
+
+## Diagnostic severity overrides
+
+Both Pylance and Pyright specify a custom [`python.analysis.diagnosticSeverityOverrides`](https://microsoft.github.io/pyright/#/configuration?id=diagnostic-rule-defaults) setting. Differences in this setting have a direct impact on the diagnostics returned. If you were attempting to get the same error output for both, making sure this setting is consistent is probably the first thing to do.
+
+
+### Diagnostic Severity Overrides to match Pyright defaults

```

Comment:
I recommend deleting this section because these defaults are the same between pyright and pylance. Including it here will confuse users more than help. If, for some reason, you think that it's important to explain the default values, I recommend simply providing a link to [pyright's documentation](https://microsoft.github.io/pyright/#/configuration?id=diagnostic-rule-defaults) rather than duplicating all of this information here. It will inevitably become stale as we add more diagnostic rules.